src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,128 @@
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { PersonNavigate } from '../PersonLink/PersonLink';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoad, setIsLoad] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [isEmpty, setIsEmpty] = useState(false);
+
+  const loadPeople = async () => {
+    setIsLoad(true);
+
+    try {
+      const peopleFromServer = await getPeople();
+
+      if (peopleFromServer.length === 0) {
+        setIsEmpty(true);
+      }
+
+      setPeople(peopleFromServer);
+    } catch (e) {
+      setIsError(true);
+    }
+
+    setIsLoad(false);
+  };
+
+  const { getSlug = '' } = useParams();
```

Comment:
```suggestion
  const { slug = '' } = useParams();
```
it's already `slug`. `getSlug` would be a good name for function that returns `slug`